stories/Atom/Lists/Lists.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './lists.scss';
+
+export const List = ({ data, type }) => {
+  const CustomTag = `${type}l`;
```

Comment:
@Shubham-Sharma-77 Instead of customtag, use variable name like listTag and for this i think we can pass complete tag like ul and ol not use only u and o